### PR TITLE
:robot: Release arm generic artifacts on release

### PR DESCRIPTION
**Pull request by Itxaka**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: .github/workflows/image-arm.yaml [hunk 1]
```diff
@@ -149,16 +149,15 @@ jobs:
       - name: Build iso  🔧
         env:
           IMAGE: "quay.io/kairos/core-${{ matrix.flavor }}-arm-generic"
-          TAG: "latest"
+          TAG: "master"
```
Comment (Itxaka) [inline .github/workflows/image-arm.yaml:152]: nicer naming for pushed artifacts

File: .github/workflows/release-arm.yaml [hunk 2]
```diff
@@ -113,3 +113,48 @@ jobs:
...
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Quay Registry
+        run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USERNAME }} --password-stdin quay.io
+      - name: Build iso  🔧
+        env:
+          IMAGE: "quay.io/kairos/core-${{ matrix.flavor }}-arm-generic"
+        run: |
+          export TAG=${GITHUB_REF##*/}
+          earthly +all-arm-generic --FLAVOR=${{ matrix.flavor }} --IMAGE=$IMAGE:$TAG --ISO_NAME=kairos-${{ matrix.flavor }}-arm-generic-$TAG
```
Comment (Itxaka) [inline .github/workflows/release-arm.yaml:146]: have to override the iso name+image name so they dont collide with the amd64 ones :/

**Comment by Itxaka** [comment]
I canceled the jobs as this doesnt affect PR pipelines

File: .github/workflows/release-arm.yaml [hunk 3]
```diff
@@ -113,3 +113,48 @@ jobs:
...
+          sudo mv build release
+      - name: Push to quay
+        env:
+          IMAGE: "quay.io/kairos/core-${{ matrix.flavor }}-arm-generic"
+          TAG: "latest"
```
Comment (mauromorales) [inline .github/workflows/release-arm.yaml:151]: this means we'll have for each release the version and latest pointing to the same artifact, right?

**Review by mauromorales — APPROVED** [review]
lgtm

File: .github/workflows/release-arm.yaml (hunk 3 above)
Comment (Itxaka) [reply .github/workflows/release-arm.yaml:151]: fuck. No, this is a leftover! It should only be the github tag, this does nothing. Good eyes!

**Pull request merged**